polygon-digital-carbon/networkConfig/mumbai.json

```diff
@@ -0,0 +1,76 @@
+{
+    "network": "mumbai",
```

Comment:
OK one minor thing: shouldn't these all be relabeled to Amoy instead of mumbai since mumbai is dead?